mongorestore/oplog_test.go

```diff
@@ -191,3 +191,47 @@ func TestOplogRestoreTools2002(t *testing.T) {
 		So(result.Failures, ShouldEqual, 0)
 	})
 }
+
+type testTable struct {
+	ns     string
+	output bool
+}
+
+func TestShouldIgnoreNamespacee(t *testing.T) {
+	testtype.SkipUnlessTestType(t, testtype.UnitTestType)
+	tests := []testTable{
+		{
+			ns:     "test.system",
+			output: false,
+		},
+		{
+			ns:     "test.system.nonsense",
+			output: false,
+		},
+		{
+			ns:     "config.system.sessions",
+			output: true,
+		},
+		{
+			ns:     "config.system.indexBuilds",
+			output: true,
+		},
+		{
+			ns:     "test.system.js",
+			output: false,
+		},
+		{
+			ns:     "test.test",
+			output: false,
+		},
+		{
+			ns:     "config.cache.any",
+			output: true,
+		},
```

Comment:
Is it worth adding one more case with the `config.cache.` prefix?

Reply:
Thanks for reviewing. I guess not much extra value to be added by one more test case.